.circleci/config.yml

```diff
@@ -145,6 +150,12 @@ workflows:
   nightly:
     jobs:
       - test_acc:
+          name: "Acceptance tests: << matrix.pattern >>"
+          matrix:
+            parameters:
+              pattern:
+                - TestAcc_Aws
+                - TestAcc_Github_
```

Comment:
I think it will be more clear if we separate naming concerned by filter with an underscore from the rest of the test name. I had planned to refactor AWS test naming in another PR. WDYT ?

I the future we may want to split AWS test too with something like `TestAcc_AWS_RunGroup1_MyTestName` or `TestAcc_AWS_EC2_ATestOfEC2Resource`

But we can totally remove the underscore, as we used it to separate `TestAcc` so far I used the same pattern.

Reply:
I'm totally good with that !